### DISCUSSION TITLE
[Question][Hw3] Issues for Z3 div binary operator

### DISCUSSION THREAD

**Discussion by PSuHyeon** [prosyslab-classroom/cs424-program-reasoning#321] [Q&A]
Hello, I have question regarding div operations. 

When I use division operand using variable denominator (i.e. when second operand of div is variable), I get error saying 'div' is uninterpreted. 

After searching for the issue I found that people are having similar issues when using div and mod operator:
[https://github.com/Z3Prover/z3/issues/3962](url)

Do we have to consider division having variable as second operand?
If yes, can you give hints for interpreting div in the case of division by variable?

Thank you in advance.

**Comment by KihongHeo** [comment]
Hi.
You can ignore such cases.

**Comment by PSuHyeon** [comment]
Thank you so much
